tests/large/test_ide.py

```diff
@@ -65,37 +64,19 @@ def test_default_eclipse_ide_install(self):
         proc = subprocess.Popen(self.command_as_list(self.exec_path), stdout=subprocess.DEVNULL,
                                 stderr=subprocess.DEVNULL)
 
-        # on 64 bits, there is a java subprocess, we kill that one with SIGKILL (eclipse isn't reliable on SIGTERM)
-        if self.arch_option == "x86_64":
-            self.check_and_kill_process(["java", self.arch_option, self.installed_path],
-                                        wait_before=self.TIMEOUT_START, send_sigkill=True)
-        else:
-            self.check_and_kill_process([self.exec_path],
-                                        wait_before=self.TIMEOUT_START, send_sigkill=True)
+        self.check_and_kill_process(["java", self.installed_path], wait_before=self.TIMEOUT_START)
         proc.wait(self.TIMEOUT_STOP)
 
         # ensure that it's detected as installed:
-        self.child = pexpect.spawnu(self.command('{} ide eclipse'.format(UMAKE)))
-        self.expect_and_no_warn("Eclipse is already installed.*\[.*\] ")
+        self.child = pexpect.spawnu(self.command('{0} ide {1}'.format(UMAKE, self.command_name)))
+        self.expect_and_no_warn("{0} is already installed.*\[.*\] ".format(self.name))
```

Comment:
Same, about {} and diff, please revert this, apart from changing the command name.

While you are doing that, as they are all on the same category, maybe add a category_name = ide and change it?